terraform/variables.tf

```diff
@@ -30,10 +30,19 @@ variable "azure_sp_credentials_json" {
   default = null
 }
 
+variable "enable_blue_green" {
+  type    = bool
+  default = false
+}
+
 variable "resource_group_name" {
   type = string
 }
 
+variable "notify_storage_account_name" {
+  default = null
```

Comment:
This isn't an optional variable so don't set a default

Reply:
fixed